spec/config.json

```diff
@@ -975,6 +975,24 @@
           "description": "Sets the refresh token hook endpoint. If set it will be called during token refresh to receive updated token claims.",
           "format": "uri",
           "examples": ["https://my-example.app/token-refresh-hook"]
+        },
+        "authorization_code_hook": {
```

Comment:
Do we need all of these config elements? Maybe it makes more sense to have one endpoint and pass the grant type and let the webhook deal with that?

Reply:
Yeah, that makes much more sense. Let me take a stab at it.

Reply:
To keep backwards compatibility we could also just add a new key "hook" and deprecate the refresh token hook at some point. That way we don't break BC?